src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/repository/CardRepository.java

```diff
@@ -8,10 +8,17 @@
 import org.springframework.data.repository.query.Param;
 import org.springframework.data.rest.core.annotation.RepositoryRestResource;
 import org.springframework.data.rest.core.annotation.RestResource;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.parameters.P;
 
 @RepositoryRestResource(excerptProjection = CardEmbedProjection.class)
 public interface CardRepository extends JpaRepository<Card, Long> {
   @RestResource(exported = false)
```

Comment:
```suggestion
```
I hope the method is not allowed when you don't export the Rest resource 😉

Reply:
Ja, aber ich hab's auch ohne das versucht, und es geht ebenfalls nicht.